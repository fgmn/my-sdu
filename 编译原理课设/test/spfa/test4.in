100
300
0
63
19
0
25
1
0
4
13
0
58
30
0
34
23
0
54
27
1
8
24
1
39
5
1
24
10
1
24
26
1
77
13
1
32
15
1
28
22
1
66
20
2
68
6
2
88
21
2
61
2
2
98
21
2
34
10
2
80
5
3
43
29
3
37
27
3
34
5
3
78
15
3
25
12
3
7
12
3
15
2
3
72
30
4
84
23
4
20
21
4
34
15
4
27
28
4
65
4
5
11
15
5
11
12
5
10
12
5
73
30
5
64
3
5
99
25
5
9
23
6
94
20
6
22
22
6
74
4
6
82
5
6
85
28
6
16
9
7
61
15
7
91
12
7
15
24
7
74
11
7
20
4
7
42
29
7
65
7
8
55
6
8
83
3
8
76
27
8
95
1
8
48
4
9
95
1
9
98
12
9
17
9
9
26
3
9
35
27
9
11
27
9
39
24
10
11
30
10
47
27
10
33
14
11
15
8
11
72
7
11
78
1
12
50
13
12
83
23
12
68
8
12
24
14
12
34
17
12
20
26
12
91
1
12
99
17
12
38
25
13
88
30
13
76
4
13
80
28
13
59
27
13
82
7
13
17
5
13
18
22
13
55
26
13
36
9
13
18
3
13
67
29
14
89
16
14
90
12
14
94
26
14
39
23
14
55
23
15
29
22
15
72
26
15
32
12
15
54
10
15
48
3
15
72
12
15
39
30
16
23
8
16
65
15
16
45
3
16
61
3
16
35
12
16
18
5
17
53
1
17
71
3
17
46
4
17
50
10
18
35
27
18
61
10
18
92
6
18
21
21
19
67
3
19
98
23
19
76
20
19
83
26
19
20
27
19
33
25
19
75
9
19
35
9
19
42
1
20
63
8
20
84
30
20
23
10
20
33
29
20
34
10
20
49
29
21
96
27
21
54
11
21
95
28
21
86
15
21
40
2
21
27
17
22
47
8
22
67
4
22
86
23
23
64
20
24
95
9
24
83
22
24
78
20
24
55
27
24
40
9
25
40
16
25
85
19
25
58
14
26
35
24
26
63
17
26
80
28
26
46
12
27
66
27
27
44
19
27
53
29
27
79
20
27
32
19
28
59
15
28
93
2
28
90
14
29
49
13
29
71
1
29
82
1
29
79
18
30
40
14
30
36
23
30
91
28
30
85
24
30
74
12
30
69
4
31
33
12
31
65
21
31
65
28
31
76
27
32
91
15
32
89
26
32
61
3
32
43
28
32
57
20
32
88
14
33
34
27
33
58
8
33
82
1
34
66
7
34
39
5
34
63
14
34
93
20
34
43
30
36
82
23
37
60
26
37
97
12
37
62
5
37
37
16
38
83
9
38
43
25
38
66
24
38
59
21
39
52
7
39
41
3
39
59
10
39
57
6
40
72
29
40
47
22
41
85
13
41
61
24
41
69
21
41
62
24
41
82
11
41
44
16
42
71
23
42
70
11
42
54
30
42
82
17
42
80
18
42
44
15
42
83
11
42
93
21
43
48
4
43
54
1
43
62
12
43
95
20
44
95
16
44
84
7
45
77
30
45
68
12
46
94
14
46
82
14
46
72
11
47
51
3
47
78
30
48
60
22
48
66
29
48
73
12
48
59
20
49
91
27
50
97
18
50
69
22
50
75
29
51
55
26
52
52
21
52
56
25
52
97
14
53
66
7
54
91
20
54
55
4
54
64
10
55
66
1
56
92
15
56
74
25
57
76
12
57
85
25
58
88
9
59
64
28
60
64
9
60
95
16
60
61
26
61
95
24
62
62
13
62
88
7
63
80
26
63
81
25
64
82
14
64
77
4
64
87
16
65
82
21
65
69
21
66
95
29
67
73
10
68
84
22
68
89
29
68
70
19
68
72
23
68
90
14
68
95
26
70
73
27
71
94
11
71
86
23
71
91
3
71
93
15
71
71
22
72
93
21
74
86
16
75
94
9
75
79
7
76
98
2
78
80
21
78
84
9
79
96
26
79
84
8
79
98
20
80
88
12
83
98
24
84
96
22
85
97
7
91
95
29
92
93
13
93
95
25
93
96
30
