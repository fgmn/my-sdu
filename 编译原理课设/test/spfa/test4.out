0
590558003
590558003
590558003
13
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
34
590558003
590558003
44
590558003
1
590558003
41
590558003
590558003
590558003
590558003
60
63
23
590558003
590558003
590558003
590558003
28
17
31
590558003
53
47
590558003
590558003
39
57
63
590558003
42
35
70
27
31
60
34
15
38
79
55
55
19
37
17
30
590558003
590558003
38
590558003
590558003
46
69
85
590558003
46
41
69
61
45
44
38
590558003
36
20
101
53
24
86
590558003
47
75
43
590558003
59
58
27
48
590558003
